internals/secrethub/service_gcp_link.go

```diff
@@ -146,6 +146,7 @@ func NewServiceGCPDeleteLinkCommand(io ui.IO, newClient newClientFunc) *ServiceG
 
 func (cmd *ServiceGCPDeleteLinkCommand) Register(r command.Registerer) {
 	clause := r.Command("delete-link", "Delete the link between a SecretHub namespace and a GCP project.")
+	clause.HelpLong("After deleting the link you cannot create new GCP service accounts in the specified namespace and GCP project anymore. Exisiting service accounts will keep on working.")
```

Comment:
So the long help will always be shown next to the 'short help'?

Reply:
Yes, the short help is what you see in the command overview of the parent commands help (`secrethub service gcp --help` in this case). In the help text of the command itself (`secrethub service gcp delete-link --help`) you have short help followed by the longer help text. You can find the complete template [here](https://github.com/secrethub/secrethub-cli/blob/v0.39.0/internals/secrethub/templates.go).